rasa/nlu/extractors/entity_synonyms.py

```diff
@@ -85,6 +85,7 @@ def replace_synonyms(self, entities):
             # need to wrap in `str` to handle e.g. entity values of type int
             entity_value = str(entity["value"])
             if entity_value.lower() in self.synonyms:
+                entity["synonym"] = entity_value.lower()
```

Comment:
If you need the information of the original text before it gets replaced, do you actually want it to be lowercase?